lib/gooddata/rest/connection.rb

```diff
@@ -94,7 +94,7 @@ def generate_string(length = ID_LENGTH)
 
         # Retry block if exception thrown
         def retryable(options = {}, &_block)
-          opts = { :tries => 17, :on => RETRYABLE_ERRORS }.merge(options)
+          opts = { :tries => 12, :on => RETRYABLE_ERRORS }.merge(options)
```

Comment:
What is the reason of decreasing this value?

Reply:
In the PR, I updated RETRY_TIME_COEFFICIENT = 2 So We need decreasing it from 17 to 12 to make sure total retry not take much time.
BTW, Default retry value is 12 https://github.com/gooddata/gooddata-ruby/blob/master/lib/gooddata/helpers/global_helpers.rb#L36 So I updated it to 12 to make sure consistency.